docs/CHANGELOG.asciidoc

```diff
@@ -8,16 +8,790 @@
+* Merge settings & mappings component template in @package {kibana-pull}128498[#128498]
+* Redesign agent flyout {kibana-pull}128381[#128381]
+* Adds a link from agent details page to agent dashboard {kibana-pull}127882[#127882]
+* Update add agent instructions in fleet managed mode for Kubernetes {kibana-pull}127703[#127703]
+* Added time_series_metric mapping for metric_type package field {kibana-pull}126322[#126322]
+* Added support for dimension field {kibana-pull}126257[#126257]
+* Refactor auto upgrade package policies logic {kibana-pull}125909[#125909]
+* Move mappings from index template to component template {kibana-pull}124013[#124013]
+
+Lens & Visualizations::
+* Adds normalize_by_unit function and option in *Lens* {kibana-pull}128303[#128303]
+* Adds suffix formatter in *Lens* {kibana-pull}128246[#128246]
+* Adds Xy gap settings in *Lens* {kibana-pull}127749[#127749]
+* Implements null instead of zero switch in *Lens* {kibana-pull}127731[#127731]
+* Adds ability to include empty rows setting for date histogram in *Lens* {kibana-pull}127453[#127453]
+* Adds support for multi rows headers for the table visualization in *Lens* {kibana-pull}127447[#127447]
+* Adds ability to open *Lens* visualizations in *Discover* from dashboards {kibana-pull}127355[#127355]
+* Auto-set exists filtering for last value in *Lens* {kibana-pull}127251[#127251]
+* Adds ability to include number of values in default terms field label in *lens* {kibana-pull}127222[#127222]
+* Adds ability to drop partial buckets option in *Lens* {kibana-pull}127153[#127153]
+* Addds ability to allow top metric for last value in *Lens* {kibana-pull}127151[#127151]
+* Improves Datatable content height with custom row height in *Lens* {kibana-pull}127134[#127134]
+* Adds ability to set legend pixel width in *Lens* {kibana-pull}126018[#126018]
+* Adds underlying data editor navigation in *Lens* {kibana-pull}125983[#125983]
+* Adds top metrics aggregation to AggConfigs, Expressions, and Visualize {kibana-pull}125936[#125936]
+* Adds the ability to detach from global time range in *Lens* {kibana-pull}125563[#125563]
+* Adds last value, min and max on dates, allow last value on ip_range, number_range, and date_range in *Lens* {kibana-pull}125389[#125389]
+* Adds version-aware sorting to data table in *Lens* {kibana-pull}125361[#125361]
+* Adds embeddable enhancement {kibana-pull}125299[#125299]
+* Cancel discarded searches in *Timelion* {kibana-pull}125255[#125255]
+* Cancel discarded searches in *TSVB* {kibana-pull}125197[#125197]
+* Adds the ability to allow users to disable auto-apply in *Lens* {kibana-pull}125158[#125158]
+* Adds Filter custom label for kibanaAddFilter in *Vega* {kibana-pull}124498[#124498]
+* Adds metric Viz config options, title position, and sizing in *Lens* {kibana-pull}124124[#124124]
+* Adds the ability to make graph edges easier to click {kibana-pull}124053[#124053]
+* Adds new autoExtendBounds setting to histogram agg config {kibana-pull}118855[#118855]
```

Comment:
That's rather technical, let's go with `Add "show empty rows" option to intervals function in Lens`